src/app/static/src/tests/components/adr/selectRelease.test.ts

```diff
@@ -0,0 +1,16 @@
+import {shallowMount} from "@vue/test-utils";
+import SelectRelease from "../../../app/components/adr/SelectRelease.vue";
+import Vuex from "vuex";
+
+describe("select release", () => {
+    
+    const getStore = () => {
+        const store = new Vuex.Store({});
+        return store;
+    }
+
+    it("renders select release", () => {
```

Comment:
Should also include a test that the action is invoked when `selectedDataset` changes (if truthy)